### PR TITLE
ISIS3 application

### PR DESCRIPTION
**Pull request by jessemapel**
This is the advisement application for the ISIS project currently under the Astrogeology Science Center.

### CODE REVIEW

**Comment by jessemapel** [comment]
As discussed in today's meeting, this will be addressed at the next TSC meeting.

File: Applications/ISIS.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+## History & Metrics
+Originating back in the 1970s, ISIS has a long history that is detailed [here](https://isis.astrogeology.usgs.gov/documents/IsisHistory/IsisHistory.html). Recently, the ISIS project has started transitioning to open source and is seeking greater community involvement.
+
+Currently, ISIS has 34 code contributors on Github. The majority of them are employees or past employees of the Astrogeology Science Center (ASC). In the last year, a small number of non-ASC personnel have contributed to the project.
+
+The ISIS software package has been downloaded from Anaconda cloud over a thousand times in the last year.
+
+## Scope
+The ISIS software project encompasses software used for photometric and photogramettric processing of planetary remote sensing data.
```
Comment (rbeyer) [inline Applications/ISIS.md:12]: photogrammetric should have the same number of 't's and 'm's: 2.

File: Applications/ISIS.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+The ISIS working group will have sole responsibility and discretion over the Planetary Software
+project in the following areas:
```
Comment (rbeyer) [inline Applications/ISIS.md:15]: So the object here is 'the Planetary Software project.'  Should that be 'the ISIS project?'

**Comment by jessemapel** [comment]
I am closing this pending the ISIS TC

**Pull request closed without merging**